pkg/webhook/server.go

```diff
@@ -92,8 +92,7 @@ func (s *Server) setCert() error {
 	// the cert provisioned.
 	webhookCert, err := s.cm.IssueCertificate(
 		s.certCommonName(),
-		certificate.Internal,
-		certificate.FullCNProvided())
+		certificate.Internal)
```

Comment:
Kubernetes webhooks [require](https://kubernetes.io/blog/2019/03/21/a-guide-to-kubernetes-admission-controllers/#tls-certificates) a specific SAN format of `service.namespace.svc` with NO trust domain attached (link mentions common name, but this was [changed](https://cloud.google.com/kubernetes-engine/docs/deprecations/webhookcompatibility) to SAN in k8s v1.23). 

I just want to make sure this change is compatible with that

Reply:
interesting, I was working in my local env but I will read up on this

Reply:
ok, looks like you are right, the tests failed with.
```
Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "osm-validator.k8s.io": failed to call webhook: Post "https://osm-validator.osm-system.svc:9093/validate?timeout=10s": x509: certificate is valid for osm-validator.osm-system.svc.cluster.local, not osm-validator.osm-system.svc
```

Looks like when I updated the webhook locally it was still using the old cert. Will drop this part of the change

Reply:
updated and left a comment for future me :-)